src/App/screens/Instructor/screens/GetPublished/components/GetPublishedSteps/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import map from 'lodash/map'
+import uniq from 'lodash/uniq'
+import compact from 'lodash/uniq'
+import isStepComplete from './utils/isStepComplete'
+import Checklist from './components/Checklist'
+import DescriptionBlock from './components/DescriptionBlock'
+
+const GetPublishedSteps = ({
+  instructor,
+  instructorLessons,
+}) => {
+
+  const instructorLessonStates = compact(uniq(map(instructorLessons.lessons, 'state')))
+
+  const steps = [
```

Comment:
I think we will be able to do some cool things with loading in content dynamically as you suggested. The instructor guide needs some TLC in the editing department. Structure, consistency, filling in the gaps. This is very high on my todo list.